src/os/net/net.js

```diff
@@ -138,7 +138,7 @@ os.net.addDefaultHandlers = function() {
   os.net.RequestHandlerFactory.addHandler(os.net.LocalFileHandler);
   os.net.RequestHandlerFactory.addHandler(os.net.SameDomainHandler);
   os.net.RequestHandlerFactory.addHandler(os.net.ExtDomainHandler);
-  os.net.RequestHandlerFactory.addHandler(os.net.ProxyHandler);
+  // No longer adds os.net.ProxyHandler here - depends on configuration
```

Comment:
The `goog.require` should be removed as well. I don't think the comment is necessary. If someone is curious where the proxy handler gets loaded, they'll find it quickly via search.